zebra/zebra_srte.c

```diff
@@ -133,6 +136,48 @@ static int zebra_sr_policy_notify_update_client(struct zebra_sr_policy *policy,
 	}
 	stream_putl(s, policy->color);
 
+	num = 0;
+	if (policy->segment_list.srv6_segs.num_segs > 0) {
+		/* Lookup table.  */
+		table = zebra_vrf_table(AFI_IP6, SAFI_UNICAST, VRF_DEFAULT);
+
+		for (rn = route_top(table); rn; rn = route_next(rn)) {
+			RNODE_FOREACH_RE_SAFE (rn, re, next) {
+				if (re->nhe->nhg.nexthop->srte_color ==
```

Comment:
invert this logic so the indent is not so deep  this is hard to read

Reply:
fixed.